website/docs/components/icon/index.md

```diff
@@ -9,8 +9,6 @@ related: ['icons/library']
 previewImage: assets/illustrations/components/icon.jpg
 navigation:
   keywords: ['icon', 'glyph', 'flight']
-status:
-  added: 4.9.0
```

Comment:
thought: I appreciate the diligence here, and I'm aware we don't have any guidance around when a status should be updated, but it feels a bit too early to remove this badge, considering we're still in the adoption/migration phase for this new component.

Reply:
That's fair, I will add it back. 

I just noticed that because so many components changed in this release there were a lot of badges everywhere.